neat_cache/lib/src/providers/resp.dart

```diff
@@ -203,7 +203,7 @@ class RespClient {
   Future<void> close({bool force = false}) async {
     _closing = true;
 
-    if (!_closing) {
```

Comment:
How was this even supposed to work before?

Reply:
I think we have never sent the `QUIT` message.